Source/WTF/wtf/Vector.h

```diff
@@ -862,6 +862,7 @@ class Vector : private VectorBuffer<T, inlineCapacity, Malloc> {
     template<typename U> ALWAYS_INLINE void append(const U* u, size_t size) { append<FailureAction::Crash>(u, size); }
     template<typename U> ALWAYS_INLINE bool tryAppend(const U* u, size_t size) { return append<FailureAction::Report>(u, size); }
     template<typename U> ALWAYS_INLINE void append(std::span<const U> span) { append(span.data(), span.size()); }
+    template<typename U> ALWAYS_INLINE void appendList(std::initializer_list<U> initializerList) { append(std::data(initializerList), initializerList.size()); }
```

Comment:
I have wanted this a few times. I am so glad you took the time to add this ability!